draft-ietf-rats-architecture.md

```diff
@@ -296,56 +296,62 @@ each router is an Attester and the main router is the Lead Attester.
-|                                                                  |
-|                       Composite Device                           |
-'------------------------------------------------------------------'
+              .-----------------------------.
+              |       Remote Verifier       |
+              '-----------------------------'
+                             ^
+                             | Evidence of
+                             | Composite Device
+                             |
+.----------------------------|----------------------------------.
+| .--------------------------|--------.                         |
+| |                   .-------------. |                         |
+| | Collecting Claims | Attesting   | |                         |
+| |         /-------->| Environment | |                         |
+| |         |         '-------------' |      .------------.     |
+| | .----------------.                |      |            |     |
+| | | Target         |                |<-----| Attester B |-.   |
+| | | Environment(s) |                |      '------------' |   |
+| | | (Optional      |                |<-------| Attester C |-. |
+| | |  Local         |                |        '------------' | |
+| | |  Verifier)     |                |<---------| ...        | |
+| | '----------------'                | Evidence '------------' |
+| |                                   | of Attesters            |
+| | Lead Attester A                   | (via Internal Links or  |
+| '-----------------------------------'  Network Connections)   |
+|                                                               |
+|                       Composite Device                        |
+'---------------------------------------------------------------'
 ~~~~
 {: #composite title="Conceptual Data Flow for a Composite Device"}
 
 In the Composite Device, each Attester generates its own Evidence by its
 Attesting Environment(s) collecting the claims from its Target Environment(s).
 The Lead Attester collects the Evidence of all other Attesters and then
-generates the Evidence of the whole Composite Attester.
-
-The Lead Attester's Attesting Environment may or may not include its own
-Verifier.
-One situation is that the Attesting Environment has no local Verifier.
-In this situation, the Lead Attesting Environment simply combines the various
+generates the Evidence of the whole Composite Device.
+
+Inside the entity of Attester, there may be a Target Environment
+that has the ability to verify the Evidence of other Attesters,
+it can be seen as an entity of Verifier and be called a local Verifier.
+The claims collected from this Target Environment (i.e., the local Verifier)
+have two types. One is about itself and will finally be used to evaluate
+the trustworthiness of the Lead Attester and itself. The other is about
+the verification results of other Attesters.
+
+One situation is that the Lead Attester has no local Verifier. In this situation, 
```

Comment:
Maybe start with: "in usage scenarios the Entity either includes a Lead Attester [btw I am still not sure, if we want to use this term] in one of its Target Environments or not." Then reverse and polish the following the following block of statements?

Reply:
I'd like to double-check whether you mean "include a Lead Attester in one of its Target Environments" or "include a Local Verifier in one of its Target Environments"?

Reply:
The reason for using the term Lead Attester is to explicitly clarify that there is a special Attester can directly communicate with the remote Verifier while other Attesters cannot.
Maybe use "Proxy Attester"?

Reply:
thank you for catching that :) In fact, what I meant to not spell wrong again in any case was:

""include a Local Verifier in one of its Target Environments"